src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/interfaces/ChanceDrop.java

```diff
@@ -0,0 +1,16 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.interfaces;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.ItemAttribute;
+
+
+public interface ChanceDrop extends ItemAttribute {
+
+    /**
+     * This field sets the % chance for an item to drop when a creature is killed.
+     */
+    int chance = 100;  
+
+    public default int getChance() {

```

Comment:
The entire point of this class is to have a chance. Why would there be a default for it? 

Remove the constant and make this a standard non-default, interface method